atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/qos/ThriftQueryWeighers.java

```diff
@@ -75,14 +80,14 @@ public QueryWeight estimate() {
             public QueryWeight weigh(T result, long timeTakenNanos) {
                 return ImmutableQueryWeight.builder()
                         .numBytes(safeGetNumBytesOrDefault(() -> bytesRead.apply(result)))
-                        .timeTakenNanos((int)timeTakenNanos)
-                        .numDistinctRows(1)
+                        .timeTakenNanos((int) timeTakenNanos)
```

Comment:
Why are we casting from long to int for nanoseconds? We’ll overflow in less than 3 seconds?!